src/lib/services/OutputProcessor.ts

```diff
@@ -79,12 +97,12 @@ export class OutputProcessor extends AsyncCreatable {
 		uxEvents.emit(eventType, constructedMessage);
 	}
 
-	private getEventsFromString(str: string): RuleEvent[] {
+	private getEventsFromString(str: string, startTag: string, endTag: string): RuleEvent[] {
 		const events: RuleEvent[] = [];
 
-		const regex = /SFDX-START(.*)SFDX-END/g;
-		const headerLength = 'SFDX-START'.length;
-		const tailLength = 'SFDX-END'.length;
+		const regex = new RegExp(`^${startTag}(.*)${endTag}`, 'g');
+		const headerLength = startTag.length;
+		const tailLength = endTag.length;
```

Comment:
Steps are the same as regular CliMessager updates, except we allow different tags to mean different things.